libraries/Microsoft.Bot.Builder.LanguageGeneration/LGFile.cs

```diff
@@ -5,28 +5,28 @@ namespace Microsoft.Bot.Builder.LanguageGeneration
     /// <summary>
     /// Here is a data model that can easily understanded and used as the LG file definition.
     /// </summary>
-    public class LGFile
+    public class LGSource
```

Comment:
I think this file name should also be changed to LGSource.cs to align with the class name